pkg/client/listers/extensions/v1beta1/daemonset_expansion.go

```diff
@@ -76,3 +78,38 @@ func (s *daemonSetLister) GetPodDaemonSets(pod *v1.Pod) ([]*v1beta1.DaemonSet, e
 
 	return daemonSets, nil
 }
+
+// GetHistoryDaemonSets returns a list of DaemonSets that potentially
```

Comment:
It doesn't seem that you care about the list, return directly the CR that matches the controller ref.